.travis.yml

```diff
@@ -76,7 +76,7 @@ jobs:
       os: linux
       dist: xenial
       language: python
-      python: "3.7"
```

Comment:
i think leave it as it. stick with 3.7 and upgrade in next PR if we need to do it.